src/BulletSoftBody/btSoftBody.cpp

```diff
@@ -3681,8 +3698,8 @@ void btSoftBody::defaultCollisionHandler(const btCollisionObjectWrapper* pcoWrap
                     docollideFace.psb = this;
                     docollideFace.m_colObj1Wrap = pcoWrap;
                     docollideFace.m_rigidBody = prb1;
-                    docollideFace.dynmargin = basemargin + timemargin;
-                    docollideFace.stamargin = basemargin;
+					docollideFace.dynmargin = 0.05*(basemargin + timemargin);
```

Comment:
Since it is still in research stage, I'll LGTM, , Do we want to expose those constants?

Reply:
Yes, eventually we do. But I'm still trying to understand why a smaller margin for face than for nodes helps with reducing stickiness.